serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -68,7 +68,7 @@ COPY distribution[s]/ ./
 RUN mv *.deb djl-serving_all.deb || true
 
 # Install CUDNN 8
-RUN apt-get update && apt-get install -y --no-install-recommends libcudnn8 && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-recommends libcudnn9-cuda-12 && rm -rf /var/lib/apt/lists/*
```

Comment:
Why not just use `12.4.1-cudnn-devel-ubuntu22.04` image directly?

Reply:
it also installed libcudnn9-dev-cuda-12, which is around 1GB we don't need that